t/ui/27-plugin_obs_rsync.t

```diff
@@ -45,21 +45,55 @@ $t->get_ok('/login');
 
 BAIL_OUT('Login exit code (' . $t->tx->res->code . ')') if $t->tx->res->code != 302;
 
-$t->get_ok('/admin/obs_rsync')->status_is(200, "index status")->element_exists('a[href*="/admin/obs_rsync/Proj1"]')
-  ->content_unlike(qr/script\<\/a\>/);
+sub _el {
+    my ($project, $run, $file) = @_;
+    return 'a[href="/admin/obs_rsync/' . "$project/runs/$run/download/$file" . '"]' if $file;
+    return 'a[href="/admin/obs_rsync/' . "$project/runs/$run" . '"]'                if $run;
+    return 'a[href="/admin/obs_rsync/' . $project . '"]';
```

Comment:
The `qq` operator works here just as well, there is no need for `.`.

Reply:
Like this `qq(a[href="/admin/obs_rsync/$project"])`? I just had to read up on what you meant by meant. This is one of the things that still confuse me with Perl, because there's no analogue in other languages and it looks like a function when it's really just a string literal.

Reply:
Yes, but you would more commonly see `qq{a[href="/admin/obs_rsync/$project"]}`. It's very useful but also one of the reasons Perl syntax highlighting is so hard.

Reply:
`qq` is one of the things I'm missing in other languages